package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auspice",
-  "version": "2.17.4",
+  "version": "2.18.3",
```

Comment:
This change shouldn't be necessary. I think it will go away if you rebase your fork on nextstrain:master - would you mind doing that before we merge?